core/data.go

```diff
@@ -220,11 +220,11 @@ type Blob struct {
 	Data          []byte
 }
 
-// BlobAuthHeader contains the data that a user must sign to authenticate a blob request.
+// BlobHeader contains the data that a user must sign to authenticate a blob request.
 // Signing the combination of the Nonce and the BlobCommitments prohibits the disperser from
 // using the signature to charge the user for a different blob or for dispersing the same blob
 // multiple times (Replay attack).
-type BlobAuthHeader struct {
+type BlobHeader struct {
 	// Commitments
 	encoding.BlobCommitments `json:"commitments"`
```

Comment:
For the purpose of auth, is the kzg commitment itself sufficient (i.e. the 3 length related fields are not needed)?

Reply:
I think the length is needed for payment purposes, but the length proof can probably be ommitted. I've made some updates to the document: https://www.notion.so/eigen-labs/Unified-blob-addressal-in-EigenDA-ae426c610f9e4e61b2e46eec76f7ca4d?pvs=4#ed6c05cfb2e34e3e9289a998cb25dc4d